itou/www/signup/views.py

```diff
@@ -221,18 +221,10 @@ def setup(self, request, *args, **kwargs):
         super().setup(request, *args, **kwargs)
 
     def dispatch(self, request, *args, siae_id, **kwargs):
-        # TODO(FF): After a week on production (magic link validity), remove
-        # fallback logic to decode the siae_id and specify the parameter as
-        # siae_id:int in the urls.
         try:
-            siae_id = int(siae_id)
-        except ValueError:
-            siae_id = urlsafe_base64_decode(siae_id)
-            try:
-                siae_id = int(siae_id)
-            except ValueError:
-                siae_id = None
-        self.siae = None if siae_id is None else Siae.objects.active().filter(pk=siae_id).get()
+            self.siae = Siae.objects.active().get(pk=siae_id)
+        except Siae.DoesNotExist:
+            self.siae = None
```

Comment:
```suggestion
        self.siae = Siae.objects.active().filter(pk=siae_id).first()
```
ne te plait pas ? :stuck_out_tongue:

Reply:
Non, je n’aime pas les `.first()` quand il n’y a qu’une seule ligne possible.